pipelines/shopgate.user.loginUser.v1.json

```diff
@@ -75,22 +62,15 @@
       {
         "type": "extension",
         "id": "@shopgate/shopify-user",
-        "path": "@shopgate/shopify-user/user/token/saveCustomerAccessToken.js",
```

Comment:
One unit test still seems to reference the old file.

```
 Exception during run: Error: Cannot find module '../../../../user/token/saveCustomerAccessToken'
Require stack:
- /home/user/projects/connect/extension-development/extensions/ext-shopify-user/extension/test/unit/user/token/saveCustomerAccessToken-test.js
```

Reply:
That and a whole lot more tests were failing. I removed those of the heavily refactored classes (API classes, mostly) and fixed some of the step tests that were missing mocks.